src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -18,10 +19,13 @@ export interface Props {
   className?: string;
   onDelete?: () => void;
   onReply?: () => void;
+  readBy?: string[];
+  recipientType: "student" | "guardian";
   repliesDisabledMsg?: string;
   senderIcon: React.ReactNode;
   senderName: string;
   sentAtTimestamp: Date;
+  userType: "teacher" | "student" | "guardian";
```

Comment:
So instead of passing in `userType`, we can just pass in `readBy` to indicate that we should show readReceipts. If we don't pass in `readBy`, then we don't show `readReceipts`. Plus this means that in the future if we wanted to show read receipts for students or guardians, we wouldnt need to make any changes to the component itself, the consumer would just need to start passing in `readBy`. So we could remove the `userType` prop and just rely on `readBy`